src/components/input/inputContainer/inputContainer.tsx

```diff
@@ -43,23 +44,26 @@ export const InputContainer: React.FC<IInputContainerProps> = (props) => {
         children,
         className,
         id,
+        wrapperClassName,
+        ...otherProps
     } = props;
 
     const processedVariant = isDisabled ? 'disabled' : variant;
     const containerClasses = classNames(
-        'flex h-12 w-full flex-row items-center', // Layout
+        'flex min-h-12 w-full flex-row items-center', // Layout
```

Comment:
Can we add an overflow-auto here so that the scrollbar is clipped (rounded)?

Reply:
Sounds good, I'm adding it to the Textarea wrapper 👌🏻